app/controllers/webhooks_controller.rb

```diff
@@ -69,7 +69,11 @@ def email_params
   end
 
   def authorized?
-    params[:auth].present? && params[:auth].secure_compare(ENV['WEBHOOK_AUTH_KEY'])
+    params[:auth].present? &&
```

Comment:
Is it important to check whether `params[:auth]` is supplied but an empty string? If we're only checking so that we don't blow up on nil, perhaps `params.key?(:auth)` is better.